bootstrap/Makefile.bootstrap

```diff
@@ -2,19 +2,69 @@ export SHELL = /bin/bash
 export TARDIGRADE_CI_ORG ?= plus3it
 export TARDIGRADE_CI_PROJECT ?= tardigrade-ci
 export TARDIGRADE_CI_BRANCH ?= $(or $(shell grep 'FROM $(TARDIGRADE_CI_ORG)/$(TARDIGRADE_CI_PROJECT)' Dockerfile | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' 2> /dev/null),master)
-export TARDIGRADE_CI_PATH ?= $(shell until [ -d "$(TARDIGRADE_CI_PROJECT)" ] || [ "`pwd`" == '/' ]; do cd ..; done; pwd)/$(TARDIGRADE_CI_PROJECT)
 
--include $(TARDIGRADE_CI_PATH)/Makefile
+TARDIGRADE_CI_PATH ?= $(shell until [ -d "$(TARDIGRADE_CI_PROJECT)" ] || [ "`pwd`" == '/' ]; do cd ..; done; pwd)/$(TARDIGRADE_CI_PROJECT)
+TARDIGRADE_CI_PATH_LOCAL := $(PWD)/$(TARDIGRADE_CI_PROJECT)
+TARDIGRADE_CI_BOOTSTRAP := $(PWD)/.$(TARDIGRADE_CI_PROJECT)
+TARGIDRADE_CI_CLONE_URL ?= https://github.com/$(TARDIGRADE_CI_ORG)/$(TARDIGRADE_CI_PROJECT).git
+
+# Resolves to TARDIGRADE_CI_PATH_LOCAL when TARDIGRADE_CI_PATH does not exist
+export TARDIGRADE_CI_PATH := $(or $(wildcard $(TARDIGRADE_CI_PATH)),$(TARDIGRADE_CI_PATH_LOCAL))
+TARDIGRADE_CI_MAKEFILE := $(TARDIGRADE_CI_PATH)/Makefile
+
+# Macro to clone TARDIGRADE_CI_PROJECT
+# editorconfig-checker-disable
+define $(TARDIGRADE_CI_PROJECT)/clone
+if [[ -d "$(TARDIGRADE_CI_PATH_LOCAL)" ]]; then \
+  echo "Removing existing $(TARDIGRADE_CI_PATH_LOCAL)" >&2 &&\
+  rm -rf "$(TARDIGRADE_CI_PATH_LOCAL)" ;\
+fi && \
+echo "Cloning $(TARGIDRADE_CI_CLONE_URL)?ref=$(TARDIGRADE_CI_BRANCH)..." >&2 &&\
+git clone -c advice.detachedHead=false --depth=1 -b "$(TARDIGRADE_CI_BRANCH)" "$(TARGIDRADE_CI_CLONE_URL)"
```

Comment:
TARGIDRADE_CI_CLONE_URL is consistently misspelled, so it works, but it might be worth correcting because my assumption was that this was an error until I saw it elsewhere.

Reply:
haha good catch, a typo, plus copy pasta of course.